pruning/manager_test.go

```diff
@@ -0,0 +1,274 @@
+package pruning_test
+
+import (
+	"container/list"
+	"fmt"
+
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/pruning"
+	"github.com/cosmos/cosmos-sdk/pruning/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/tendermint/libs/log"
+	db "github.com/tendermint/tm-db"
+)
+
+func Test_NewManager(t *testing.T) {
+	manager := pruning.NewManager(log.NewNopLogger())
+
+	require.NotNil(t, manager)
+	require.NotNil(t, manager.GetPruningHeights())
+	require.Equal(t, types.PruningNothing, manager.GetOptions().GetPruningStrategy())
+}
+
```

Comment:
Please also add a security related test where you mess with the database and assign a negative height to the database then try to load that height and ensure it fails with an error, otherwise the current code has this attack vector and no test for it.